resources/views/layouts/participant_master.blade.php

```diff
@@ -10,6 +10,7 @@
     <link href="https://fonts.googleapis.com/css?family=Raleway:100,600" rel="stylesheet" type="text/css">
     <!-- Bootstrap CSS -->
     <link rel="stylesheet" href="{{ mix('css/stb_bundle.css') }}">
+    <link rel="stylesheet" href="{{ mix('css/welcome.css') }}">
```

Comment:
Dit is overbodig indien je jouw .less file hebt toegevoegd aan mix.styles voor de participant in de webpack.mix.js